shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -1031,6 +1038,12 @@ object GlutenConfig {
       .checkValue(_ > 0, s"$GLUTEN_MAX_BATCH_SIZE_KEY must be positive.")
       .createWithDefault(4096)
 
+  val GLUTEN_COLUMNAR_TO_ROW_MEM_THRESHOLD =
+    buildConf(GLUTEN_COLUMNAR_TO_ROW_MEM_THRESHOLD_KEY)
+      .internal()
+      .longConf
+      .createWithDefault(256 * 1024 * 1024)
+
```

Comment:
we have support the usage of `set spark.gluten.sql.columnarToRowMemoryThreshold=256m`